concent_api/command_line_tool/key_manager.py

```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+from utils.testing_helpers import generate_ecc_key_pair
+
+REQUESTOR_PRIVATE_KEY = b"\xf1)\x82\xfd\xa7t\xd5\x7f\x00K\xa9\x12\x80\xed\x88\xef\xae\x03;\x11<0\xf8b\x96@\xaa\x92\xef\xa6;'"
```

Comment:
Why are those hardcoded ? Shouldn't they be generated ?

Reply:
They are generated when not hardcoded. Now they are hardcoded to reuse them for receiving messages form receive endpoint.

Reply:
When all set to None they will be generated e.g.
REQUESTOR_PRIVATE_KEY = None
REQUESTOR_PUBLIC_KEY = None

PROVIDER_PRIVATE_KEY = None
PROVIDER_PUBLIC_KEY = None

Reply:
Ok, I am just trying to understand it correctly. So this will work as some kind of settings, where you can use predefined keys or generated ?

Reply:
Yes, that's right.